### ISSUE TITLE
Can we directly link to MAAP datasets in EDSC?

### ISSUE THREAD

**Issue by grallewellyn** [MAAP-Project/ZenHub#944]
Question for the data team

**Comment by wildintellect** [comment]
@grallewellyn I think we need more details, does @anilnatha recent inclusion of the https://stac-browser.maap-project.org in the MAAP portal meet this need?

**Comment by grallewellyn** [comment]
@gchang Did you mean:
1. Add MAAP datasets to https://search.earthdata.nasa.gov/search if we have permission to do that
or 
2. Add something like https://stac-browser.maap-project.org/ to the EDSC dropdown?

**Comment by gchang** [comment]
I was thinking #2. We have no control over #1.

**Comment by grallewellyn** [comment]
Is https://stac-browser.maap-project.org/ the link we want to add then? And then title the dropdown item "MAAP Datasets"?

**Comment by wildintellect** [comment]
@grallewellyn can you clarify, where this proposed link would go? Is this in the ADE jupyter extensions, or somewhere on one of the websites like maap-project.org

cc: @heidimok @aboydnw

**Comment by anilnatha** [comment]
> Is https://stac-browser.maap-project.org/ the link we want to add then? And then title the dropdown item "MAAP Datasets"?

it's been about a month I think, but I had added this link to our Portal in DIT under the "Explore" menu, with a preliminary title of "STAC Browser".

**Comment by grallewellyn** [comment]
@wildintellect I was referring to the ADE Jupyter extensions so it would be added as a dropdown under the "Data Search" menu
<img width="698" alt="Screenshot 2024-05-08 at 9 20 49 AM" src="https://github.com/MAAP-Project/Community/assets/114033465/e4baf451-c386-4ac7-83fc-f851eeb301e4">
Should we add it to both @anilnatha?

**Comment by anilnatha** [comment]
@grallewellyn I think so, if @wildintellect agrees.

**Comment by wildintellect** [comment]
I'd like @heidimok @aboydnw to consider this in their website overhaul proposal. It seem if we want parity with EarthData Search we'd want more than just a link to the STAC browser, but that might be a good start.

**Comment by aboydnw** [comment]
Earthdata search shows records that can be found in NASA CMR, right? 

And MAAP users will want to go to earthdata search to find available data, and then what will they do once they find the data they need?

Sorry, I am a little ignorant on this topic so I'm trying to understand the user needs as well as technical implications

**Comment by wildintellect** [comment]
@aboydnw it's complicated https://docs.maap-project.org/en/latest/technical_tutorials/search/catalog.html because there are 2 sources for searching for data. Once they find the data, then they'll want to convert those searches to code in their notebooks. Let me know if you need more explanation.

**Comment by aboydnw** [comment]
@wildintellect makes sense. I'm a little unfamiliar with the actual user journey for converting searches to code. Is that workflow fairly baked? Or is there potential improvements that the Dashboard could help facilitate?

**Comment by wildintellect** [comment]
@aboydnw it's only lightly documentated https://docs.maap-project.org/en/latest/system_reference_guide/faq/copy_earthdata.html
but the idea is similar to what VEDA has been trying to do where you can launch a notebook from a Dashboard/Data Search, and have the code set for the same dataset.

**Comment by aboydnw** [comment]
Unless we want to go down the path of showing data from earthdata search in the VEDA catalog, which would be more of an integration question than a high level design question, then I think this is just another link in the nav for the proposed MAAP Dashboard. 

So we could send people to the MAAP Data Catalog (records in STAC, shown through VEDA’s native data catalog) or we can send people to earthdata search and they can do what they already do to find data and pull it into the ADE portal (if I’m understanding that workflow correctly)

My guess is that these users already have a well worn path to earthdata search, so we shouldn't invest too much energy into this connection unless there is value in a tighter integration where they only ever need to look for data in the native VEDA data catalog
